monkey/monkey_island/cc/models/attack_mitigation.py

```diff
@@ -0,0 +1,23 @@
+from mongoengine import Document, StringField, DoesNotExist
+
+
+class AttackMitigation(Document):
+
+    technique_id = StringField(required=True, primary_key=True)
+    name = StringField(required=True)
+    description = StringField(required=True)
+
+    @staticmethod
+    def get_mitigation_by_technique_id(technique_id: str) -> Document:
+        try:
+            return AttackMitigation.objects.get(technique_id=technique_id)
+        except DoesNotExist:
+            raise Exception("Attack technique with id {} does not exist!".format(technique_id))
```

Comment:
1. Too broad exception type, might be better to keep the DoesNotExist or create one of your own
2. You're not catching this exception so if a mitigation exists the report gets stuck - it should show "mitigation not found" instead.

Reply:
1. I don't really care about exception type, because we add a message that tells exactly what's happening. 
2. This method shouldn't be called with invalid ID. If it's called with invalid ID, island should crash so that dev notices it and fixes.